restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/ConferencesEndpoint.java

```diff
@@ -194,4 +202,75 @@ protected Response getConferences(final String accountSid, UriInfo info, MediaTy
+            secure(account, "RestComm:Modify:Conferences");
+        } catch (final AuthorizationException exception) {
+            return status(UNAUTHORIZED).build();
+        }
+        final ConferenceDetailRecordsDao dao = daoManager.getConferenceDetailRecordsDao();
+        ConferenceDetailRecord cdr = dao.getConferenceDetailRecord(new Sid(sid));
+        if (cdr == null) {
+            return status(NOT_FOUND).build();
+        } else {
+            //allow super admin to perform update on any conference - https://telestax.atlassian.net/browse/RESTCOMM-1171
+            if (!isSuperAdmin()) {
+                try {
+                    secure(account, cdr.getAccountSid(), SecuredType.SECURED_STANDARD);
+                } catch (final AuthorizationException exception) {
+                    return status(UNAUTHORIZED).build();
+                }
+            }
+
+            final String status = data.getFirst("Status");
+
+            if(status != null){
+                if (status.equalsIgnoreCase("completed")) {
+                    kickoutAllActiveParticipants(cdr, effectiveAccount);
+                    //get updated conference record
+                    cdr = dao.getConferenceDetailRecord(new Sid(sid));
+                }else {
+                    return status(BAD_REQUEST).build();
+                }
+            }else {
+                    return status(BAD_REQUEST).build();
+            }
+
+            if (APPLICATION_XML_TYPE == responseType) {
+                final RestCommResponse response = new RestCommResponse(cdr);
+                return ok(xstream.toXML(response), APPLICATION_XML).build();
+            } else if (APPLICATION_JSON_TYPE == responseType) {
+                return ok(gson.toJson(cdr), APPLICATION_JSON).build();
+            } else {
+                return null;
+            }
+        }
+    }
+
+    private void kickoutAllActiveParticipants(ConferenceDetailRecord conferenceDetailRecord, Account effectiveAccount){
+        List<CallDetailRecord> callDetailRecords = daoManager.getCallDetailRecordsDao().getRunningCallDetailRecordsByConferenceSid(conferenceDetailRecord.getSid());
+
+        if(callDetailRecords == null || callDetailRecords.isEmpty()){
+            if (logger.isDebugEnabled())
+                logger.debug("no active participants found.");
+        } else {
+           try {
+               if (logger.isDebugEnabled())
+                    logger.debug("total conference participants are: "+callDetailRecords.size());
+               Iterator<CallDetailRecord> iterator = callDetailRecords.iterator();
+                while(iterator.hasNext()){
+                    final CallDetailRecord CallDR = iterator.next();
+                    if (callManager == null)
+                        callManager = (ActorRef) context.getAttribute("org.restcomm.connect.telephony.CallManager");
+                    callManager.tell(new Hangup("Conference Terminated", effectiveAccount.getSid(), CallDR), null);
```

Comment:
@maria-farooq Does it worth to create a new message instead of use Hangup that is used by VoiceInterpreter to ask the Call actor to terminate the call? 
Not important, just wondering if it would be a better approach to have a new message here